packages/editor-ui/src/components/__tests__/__snapshots__/ChangePasswordModal.test.ts.snap

```diff
@@ -0,0 +1,6 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`ChangePasswordModal > should render correctly 1`] = `
+"<!--teleport start-->
+<!--teleport end-->"
+`;
```

Comment:
This snapshot is not very useful, and I couldn't get the test to render the actual component. I still left it so we test that rendering at least works. I can also remove it if better that way